web/flux_to_redux/advanced/examples/influx_implementation/store.dart

```diff
@@ -0,0 +1,154 @@
+        _actions.changeBlockTwoBackgroundColor, (_) => this.influxReducer(UpdateBlockTwoBackgroundColorAction()));
+  }
+}
+
+class AnotherReduxState {
+  String backgroundColor;
+
+  AnotherReduxState.defaultState() : this.backgroundColor = 'Blue';
+
+  AnotherReduxState.from(AnotherReduxState oldState, {String backgroundColor})
+      : this.backgroundColor = backgroundColor ?? oldState.backgroundColor;
+}
+
+class AnotherColorStore extends flux.Store with InfluxStoreMixin<AnotherReduxState> {
+  @override
+  get reduxReducer => anotherReduxReducer;
+
+  RandomColorActions _actions;
+
+  /// Public data
+  String get backgroundColor => state.backgroundColor;
+
+  AnotherColorStore(this._actions) {
+    state = AnotherReduxState.defaultState();
+    triggerOnActionV2(
+        _actions.changeBlockThreeBackgroundColor, (_) => this.influxReducer(UpdateBlockThreeBackgroundColorAction()));
+  }
+}
+
+TopLevelReduxState topLevelReduxReducer(TopLevelReduxState oldState, dynamic action) {
+  if (action is UpdateBackgroundColorAction) {
+    var color = '#' + (Random().nextDouble() * 16777215).floor().toRadixString(16);
+    return TopLevelReduxState.from(oldState, mainBackgroundColor: color);
+  } else if (action is UpdateBlockOneBackgroundColorAction) {
+    var color = '#' + (Random().nextDouble() * 16777215).floor().toRadixString(16);
+    return TopLevelReduxState.from(oldState, blockOneBackgroundColor: color);
+  }
+
+  return oldState;
+}
+
+LowLevelReduxState lowLevelReduxReducer(LowLevelReduxState oldState, dynamic action) {
+  if (action is UpdateBlockTwoBackgroundColorAction) {
+    var color = '#' + (Random().nextDouble() * 16777215).floor().toRadixString(16);
+    return LowLevelReduxState.from(oldState, backgroundColor: color);
+  }
+
+  return oldState;
+}
+
+AnotherReduxState anotherReduxReducer(AnotherReduxState oldState, dynamic action) {
+  if (action is UpdateBlockThreeBackgroundColorAction) {
+    var color = '#' + (Random().nextDouble() * 16777215).floor().toRadixString(16);
+    return AnotherReduxState.from(oldState, backgroundColor: color);
+  }
+
+  return oldState;
+}
+
+RandomColorActions randomColorActions = RandomColorActions();
+RandomColorStore randomColorStore = RandomColorStore(randomColorActions);
+LowLevelStore lowLevelStore = LowLevelStore(randomColorActions);
+AnotherColorStore anotherColorStore = AnotherColorStore(randomColorActions);
+
+// Create a context instance for each new store instance
+Context randomColorStoreContext = createContext();
+Context lowLevelStoreContext = createContext();
+Context anotherColorStoreContext = createContext();
+
+// Create an adapter store for each new store instance
+FluxToReduxAdapterStore randomColorStoreAdapter =
+    randomColorStore.asReduxStore(RandomColorActions);
+FluxToReduxAdapterStore lowLevelStoreAdapter =
+    lowLevelStore.asReduxStore(RandomColorActions);
+FluxToReduxAdapterStore anotherColorStoreAdapter =
+    anotherColorStore.asReduxStore(RandomColorActions);
```

Comment:
These are causing a RTE in this example
> Uncaught Error: Expected a value of type 'RandomColorActions', but got one of type '_Type'

and should be `randomColorActions` instead of `RandomColorActions`.